torchok/metrics/detection.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict, List
+
+import numpy as np
+from mmdet.core.evaluation.mean_ap import eval_map
+from torch import Tensor
+from torchmetrics import Metric
+
+from torchok.constructor import METRICS
+
+
+@METRICS.register_class
+class MMDetectionMAP(Metric):
```

Comment:
Has it problems with memory?

Reply:
The problem with memory is in another metric `CocoEvalMAP`. But this problem may be reduced by using `compute_on_step: false` parameter